File: uri/python/1040.py
# -*- coding: utf-8 -*-

primeira, segunda, terceira, quarta = input().split(" ")

primeira = float(primeira)
segunda = float(segunda)
terceira = float(terceira)
quarta = float(quarta)

media = 0
mediafinal = 0

prova = 0

media = ((primeira * 2) + (segunda * 3) + (terceira * 4) + quarta) / 10

if media >= 7: 
     print('Media: {:.1f}'.format(media))
     print('Aluno aprovado.') 
elif media < 5:
    print('Media: {:.1f}'.format(media))
    print('Aluno reprovado.')
elif media >= 5 and media < 7:
    print('Media: {:.1f}'.format(media)) 
    print('Aluno em exame.')
    prova = float(input())
    mediafinal = (prova + media) / 2
    if mediafinal >= 5:
        print('Nota do exame: {:.1f}'.format(prova))
        print('Aluno aprovado.')
        print('Media final: {:.1f}'.format(mediafinal))
    elif mediafinal < 5: 
        print('Nota do exame: {:.1f}'.format(prova))
        print('Aluno reprovado.')
        print('Media final: {:.1f}'.format(mediafinal))
        

File: uri/C/beginner/1040.c
#include <stdio.h>

int main()
{

	double N1, N2, N3, N4, a, b, c, d, media, exame, mediaE;

	scanf("%lf %lf %lf %lf", &a, &b, &c, &d);

	N1 = 2 * a;
	N2 = 3 * b;
	N3 = 4 * c;
	N4 = 1 * d;

	media = (N1 + N2 + N3 + N4)/10;
	
	printf("Media: %.1lf\n", media);

	if(media >= 7){
		printf("Aluno aprovado.\n");
	}

	if(media < 5){
		printf("Aluno reprovado.\n");
	}

	if(media >= 5 && media < 7){
		printf("Aluno em exame.\n");
		scanf("%lf", &exame);
		printf("Nota do exame: %.1lf\n", exame);
		mediaE = (media + exame)/2;
		if(mediaE >= 5){
		printf("Aluno aprovado.\n");
		}
		if(mediaE < 5){
			printf("Aluno reprovado.\n");
		}
		printf("Media final: %.1lf\n", mediaE);
	}


	return 0;
}
